dot/parachain/candidate-validation/worker.go

```diff
@@ -0,0 +1,91 @@
+package candidatevalidation
+
+import (
+	parachainruntime "github.com/ChainSafe/gossamer/dot/parachain/runtime"
+	parachaintypes "github.com/ChainSafe/gossamer/dot/parachain/types"
+)
+
+type worker struct {
+	workerID    parachaintypes.ValidationCodeHash
```

Comment:
There was a comment about changing this to parachain id, but I'd like to keep it as validation code hash since the validation is for that give instance of the code.  I'm concerned that there may be future parachain runtime updates, where parachain ID would stay the same, but the validation code may have changed.